app/src/main/java/com/simplecity/amp_library/ui/screens/songs/menu/SongMenuPresenter.kt

```diff
@@ -142,10 +139,11 @@ open class SongMenuPresenter @Inject constructor(
             ))
     }
 
-    override fun removeSong(context: Context, song: Song) {
+    override fun removeSong(mainController: MainController, context: Context, song: Song) {
         Log.d(TAG, "removeSong: here")
-        val playlistDetailFragment = (context as MainActivity)
-                .supportFragmentManager
+
+        val playlistDetailFragment = mainController
```

Comment:
Instead of saving a reference to `childFragmentManager` earlier in `MainController`, let's call `getChildFragmentManager()` directly here.

Reply:
@BumbleFlash Please see above ☝️ . Could you still make this change?

Reply:
Ah, nevermind, this is done, the code just looks the same in Kotlin because it removes the `get`.